lib/llvm-backend/src/code.rs

```diff
@@ -78,22 +78,16 @@ fn trap_if_not_representatable_as_int(
     let continue_block = context.append_basic_block(function, "conversion_success_block");
 
     let float_ty = value.get_type();
-    let (int_ty, float_ptr_ty, float_size) = if float_ty == intrinsics.f32_ty {
-        (intrinsics.i32_ty, intrinsics.f32_ptr_ty, FloatSize::Bits32)
+    let (int_ty, float_size) = if float_ty == intrinsics.f32_ty {
+        (intrinsics.i32_ty, FloatSize::Bits32)
     } else if float_ty == intrinsics.f64_ty {
-        (intrinsics.i64_ty, intrinsics.f64_ptr_ty, FloatSize::Bits64)
+        (intrinsics.i64_ty, FloatSize::Bits64)
     } else {
         unreachable!()
     };
 
     let (exponent, invalid_exponent) = {
-        let float_bits = {
-            let space = builder.build_alloca(int_ty, "space");
-            let float_ptr = builder.build_pointer_cast(space, float_ptr_ty, "float_ptr");
-            builder.build_store(float_ptr, value);
-            builder.build_load(space, "float_bits").into_int_value()
-        };
-
+        let float_bits = builder.build_bitcast(value, int_ty, "").into_int_value();
```

Comment:
Could name be something more descriptive than “” ? Might help with debugging later?

Reply:
Done.